xarray/tests/test_dataarray.py

```diff
@@ -410,6 +410,18 @@ def test_constructor_from_self_described(self):
         actual = DataArray(IndexVariable("foo", ["a", "b"]))
         assert_identical(expected, actual)
 
+    @requires_dask
+    def test_constructor_from_self_described_chunked(self):
+        expected = DataArray(
+            [[-0.1, 21], [0, 2]],
+            coords={"x": ["a", "b"], "y": [-1, -2]},
+            dims=["x", "y"],
+            name="foobar",
+            attrs={"bar": 2},
+        ).chunk()
+        actual = DataArray(expected)
+        assert_identical(expected, actual)
```

Comment:
This does not check if the chunks are equal.
```suggestion
        assert_chunks_equal(expected, actual)
```

Reply:
True, but shouldn't we check both, that the array is identical and the chunks are the same? If I see correctly, `assert_chunks_equal` does not compare assigned attributes, like `coords`, `attrs`.

Reply:
Yes checking both is good.

Reply:
alright, both tests are there now